### PR TITLE
added gci student rutuja and mentor Minal

### PR DESCRIPTION
**Pull request by wavicles**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://wavicles.github.io/gci15.fossasia.org

**Comment by mananwason** [comment]
@wavicles please solve the conflicts. Then we can merge.

**Comment by mananwason** [comment]
[This](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) might help.

File: _data/students.yml [hunk 1]
```diff
@@ -680,7 +680,7 @@
 
 - name: Yasmin
   github: yasmin22 
-  image: ravina.jpg
+  image: yasmin.jpg
```
Comment (jigyasa-grover) [inline _data/students.yml:683]: This has been corrected in PR #690 , Hence the conflict.

**Comment by wavicles** [comment]
Thanks @mananwason @jig08 
Too many things to learn :(

**Comment by jigyasa-grover** [comment]
@wavicles  :)

**Comment by mananwason** [comment]
No issues @wavicles. :smile:

**Comment by mananwason** [comment]
You can solve the conflicts in this PR itself @wavicles

**Pull request closed without merging**